src/containers/Storage/Storage.js

```diff
@@ -201,6 +206,15 @@ class Storage extends React.Component {
         setStorageType(value);
     };
 
+    onUptimeFilterChange = (value) => {
+        this.props.setNodesUptimeFilter(value);
+    };
+
+    onShowAllNodes = () => {
```

Comment:
Here I messed up with fixups, this change relates to next commit (fix(Storage): add message on empty nodes with small uptime)